### PR TITLE
Enhancement/short methods sig

### PR DESCRIPTION
**Pull request by ric-jon**
More classes fixed

### CODE REVIEW

**Comment by dvantetering** [comment]
Nice work!

File: src/main/java/core/model/Model.java [hunk 1]
```diff
@@ -200,38 +200,29 @@ public void setAnnotations(List<Annotation> annotations) {
      */
     public Boolean addCell(int id, String text, int nucleotides, CellType type) {
         switch (type) {
-            case RECTANGLE:
-                RectangleCell rectangleCell = new RectangleCell(id, nucleotides);
+            case RECTANGLE: RectangleCell rectangleCell = new RectangleCell(id, nucleotides);
```
Comment (arbreurkes) [inline src/main/java/core/model/Model.java:203]: **NOTE:** Auto-format in IntelliJ IDEA will wreck this. Watch out with using it.

File: src/main/java/core/parsers/AnnotationParser.java [hunk 2]
```diff
@@ -27,7 +27,6 @@ private AnnotationParser() {
      * @throws IOException Throw exception on read failure.
      */
     @SuppressFBWarnings("I18N")
-    @SuppressWarnings("CheckStyle.MethodLength")
```
Comment (arbreurkes) [inline src/main/java/core/parsers/AnnotationParser.java:30]: Good point.

**Comment by arbreurkes** [comment]
This is of good use, just watch out while using any auto-format.

**Pull request merged**